CHANGES.md

```diff
@@ -64,8 +64,11 @@ This change also aligned the authentication architectures between DC/OS Enterpri
 * Admin Router returns relative redirects to avoid relying on the Host header (DCOS-47845)
 
 * Introduced the `dcos-ui-update-service`, this component exposes an API to update the servered `dcos-ui` version using the `dcos-ui` package published to Universe.
+
 * Add basic support for prometheus to dcos-net (DCOS_OSS-4738)
 
+* Add Metrics for dns forwarding (DCOS-48336)
```

Comment:
nit: Add metrics for DNS forwarding

Reply:
Thanks, I will save it for next time I will touch this file because it will happen within the next RI.